_includes/v22.2/ui/insights.md

```diff
@@ -0,0 +1,224 @@
+|`sql.insights.execution_insights_capacity`                              | `1000`        | The maximum number of execution insights stored in each node.                                                                                                                                 | Statement executions                 |
+|`sql.contention.event_store.capacity`                                   | `64 MiB`      | The in-memory storage capacity of the contention event store in each nodes.                                                                                                                   | Transaction executions               |
+|`sql.contention.event_store.duration_threshold`                         | `0`           | The minimum contention duration to cause contention events to be collected into the `crdb_internal.transaction_contention_events` table.                                                      | Transaction executions               |
+
+#### Detect slow executions
+
+There are two different methods for detecting slow executions. By default, they are both enabled and you can configure them based on your workload.
+
+The first method flags all executions running longer than `sql.insights.latency_threshold`. This is analogous to checking the [slow query log]({{ link_prefix }}logging-use-cases.html#sql_perf).
+
+The second method attempts to detect **unusually slow executions**. You can enable this detection with `sql.insights.anomaly_detection.enabled` and configure it with `sql.insights.anomaly_detection.latency_threshold`.
+CockroachDB will then keep a streaming histogram in memory for each distinct statement fingerprint that has seen an execution latency longer than `sql.insights.anomaly_detection.latency_threshold`, and will flag any execution with a latency in the 99th percentile (greater than p99) for its fingerprint.
+
+Additional controls filter out executions that are less actionable:
+
+- The execution's latency must also be longer than twice the median latency (`> 2*p50`) for that fingerprint. This ensures that the elevated latency is significant enough to warrant attention.
+- The execution's latency must also be longer than `sql.insights.anomaly_detection.latency_threshold`. Some executions are slower than usual, but are still fast enough for the workload.
+
+The [`sql.insights.anomaly_detection.memory_limit` cluster setting]({{ link_prefix }}cluster-settings.html#setting-sql-insights-anomaly-detection-memory-limit) cluster setting limits the amount of memory available for tracking these streaming latency histograms. When this threshold is surpassed, the least-recently touched histogram is evicted. The default of `1 MiB` is sufficient for tracking about 1,000 fingerprints.
```

Comment:
Suggest not repeating the default value of this cluster setting here.  The linked page is always going to be up to date.  This could say "the default setting is sufficient for tracking about 1,000 fingerprints".  Then if the default value changes it would still be wrong but at least only half as wrong :-)